### DISCUSSION TITLE
The description of generics is not detailed enough

### DISCUSSION THREAD

**Discussion by tzSharing** [vlang/v#22105] [General]
The description of generics in the V documentation is not detailed enough.Many issues are not mentioned, such as:

-  Is it type erased?

-  How to determine the type?

-  How to force type conversion?

-  How to constrain types?

**Comment by Wajinn** [comment]
> The description of generics in the V documentation is not detailed enough.

Maybe you want to do this as an issue too?

**Reply by tzSharing** [reply]
> Maybe you want to do this as an issue too?

I wasn't sure if this topic was appropriate for “issues”, so here it is.
